FLEDGE_browser_bidding_and_auction_API.md

```diff
@@ -93,3 +105,271 @@ Exposing size of the blob is a temporary leak that we hope to mitigate in the fu
+                "type": "number"
+              },
+              { "$ref": "#/$defs/adRenderId"}
+            ]
+          }
+        }
+      }
+    }
+  }
+}
+```
+
+This roughly matches the specification of the interest group in the B&A explainer.
+
+The request is framed with a 5 byte header, where the first byte is the value 0x02 and the following 4 bytes are the length of the request message in network byte order.
+Then the request is padded to a set of pre-configured lengths (TBD).
+
+### Example
+
+The JSON equivalent of an example `auctionBlob` would look like this:
+
+```json
+{
+  "version": 0,
+  "publisher": "https://foo.com",
+  "generationId": "11111111-2222-3333-4444-555555555555",
+  "interestGroups": {
+    "https://owner1.com": "<bytes>",
+    "https://owner2.com": "<bytes>"
+  }
+}
+```
+
+Where `<bytes>` are filled in with gzip compressed CBOR encoded list of interest groups for each owner.
+
+The JSON equivalent of the interest group would look like the following example:
+```json
+{
+  "name": "cars",
+  "biddingSignalsKeys": ["key1", "key2"],
+  "userBiddingSignals": "signals",
+  "ads": [
+    "adRenderId",
+    "adRenderId2"
+  ],
+  "componentAds": [],
+  "browserSignals": {
+    "joinCount": 2,
+    "bidCount": 0,
+    "recency": 500,
+    "prevWins": [
+        [20, "adRenderId"],
+        [100, "adRenderId"]
+      ]
+  }
+}
+```
+
+## Response Blob Format
+
+The response blob from a B&A auction contains an HPKE encrypted AuctionResult. This response has an encryption header like that
+used in OHTTP and serves as the response for the encryption context started by the `auctionBlob` from `navigator.getInterestGroupAdAuctionData`.
+The response contains a framing header like the request and contains a blob of compressed data, using the same schema version and same compression algorithm as specified in the `auctionBlob`.
+The response needs to be padded to a set of sizes to limit the amount of information leaking from the auction.
+
+Prior to compression and encryption, the AuctionResult is encoded as CBOR with the following schema (specified using [JSON Schema](https://datatracker.ietf.org/doc/html/draft-bhutton-json-schema-01)):
+
+```json
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "/SellerFrontEnd/AuctionResponse",
+  "type": "object",
+  "$defs": {
+    "errorDef": {
+      "type": "object",
+      "properties": {
+        "code": {
+          "description": "HTTP status code for the request.",
+          "type": "number"
+        },
+        "message": {
+          "description": "Human readable error message.",
+          "type": "string"
+        }
+      },
+      "additionalProperties": false
+    },
+    "reportingUrlsDef": {
+      "type": "object",
+      "properties": {
+        "reportingUrl": {
+          "type": "string",
+          "format": "uri"
+        },
+        "interactionReportingUrls": {
+          "description": "Map from events to beacon URLs",
```

Comment:
can you add a link or a reference to the Fenced Frame Ads Reporting API to explain what "events" and "beacon URLs" are?